test/functional/change_stream.test.js

```diff
@@ -1982,7 +1982,7 @@ describe('Change Streams', function() {
       }
     });
 
-    it('when invoked with callbacks', {
+    it.skip('when invoked with callbacks', {
```

Comment:
These two tests fail; the callbacks-based one consistently, and the promises-based on intermittently, which is a little concerning, but I'm not 100% they are written correctly.

They were already flaky, e.g. [see here](https://evergreen.mongodb.com/task/mongo_node_driver_stable_rhel70_boron_test_4.0_replica_set_unified_8a69ed696f4bf8d8790874118b72518d366b8fad_20_04_23_18_46_05).

I'd propose fixing them (if desired) be moved into a separate ticket.

Reply:
SGTM